navigation/components/BackButtonHeader/styles.js

```diff
@@ -0,0 +1,9 @@
+import { StyleSheet } from 'react-native';
+
+export const styles = StyleSheet.create({
+  image: {
+    margin: 0,
+    width: 20,
+    height: 20
+  }
+});
```

Comment:
I'm not really sure this file is necessary, can you use `headerBackTitleStyle` in `AppNavigation.js` and see if the back button looks the same?

Reply:
This prop doesnt work for me, i need these styles by reduce the image size. :disappointed: